spec/OBD/OBD.vspec

```diff
@@ -34,7 +47,7 @@
   description: Malfunction Indicator Light (MIL) False = Off, True = On
 
 - Status.DTCCount:
-  datatype: uint32
+  datatype: uint8
```

Comment:
As discussed in previous VSS-meeting  - keeping int-based values for PID representing counts. But then it makes sense to select the "right" VSS size. In this case OBD use 7 bits (see https://en.wikipedia.org/wiki/OBD-II_PIDs#Service_01_PID_01) so uint8 shall suffice, no need to have uint32